packages/server/utils/OpenAIServerManager.ts

```diff
@@ -17,6 +17,39 @@ class OpenAIServerManager {
     this.openAIApi = new OpenAIApi(configuration)
   }
 
+  async getStandupSummary(htmlResponses: string[], meetingPrompt: string) {
+    if (!this.openAIApi) return null
+    // :TODO: (jmtaber129): Include info about who made each response in the prompt, so that the LLM
+    // can include that in the response, e.g. "James is working on AI Summaries" vs. "Someone is
+    // working on AI Summaries".
+    const prompt = `Below is a list of responses submitted by team members to the question "${meetingPrompt}". If there are multiple responses, the responses are delimited by the string "NEW_RESPONSE". Identify up to 5 themes found within the responses. For each theme, provide a 2 to 3 sentence summary. In the summaries, only include information specified in the responses. When referring to people in the output, do not assume their gender and default to using the pronouns "they" and "them".
+
+    Desired format:
+    * <theme title>: <theme summary>
+    * <theme title>: <theme summary>
+    * <theme title>: <theme summary>
+
+    Responses: """
+    ${htmlResponses.join('\nNEW_RESPONSE\n')}
+    """`
+    try {
+      const response = await this.openAIApi.createCompletion({
+        model: 'text-davinci-003',
```

Comment:
-1 davinci is the older model that Bruce is currently replacing with gpt-3.5-turbo here https://github.com/ParabolInc/parabol/pull/8227. That model is faster & cheaper, so we should probably start with it here or we'll need to replace it soon anyway.

Reply:
Fixed!